base/complex.jl

```diff
@@ -535,37 +535,39 @@ end
 #     return Complex(abs(iz)/r/2, copysign(r,iz))
 # end
 
-# compute exp(im*theta)
-function cis(theta::Real)
-    s, c = sincos(theta)
-    Complex(c, s)
-end
-
 """
-    cis(z)
+    cis(x)
 
-Return ``\\exp(iz)``.
+Same as `exp(im*x)`, this computes ``\\exp(i x)``, but is typically faster.
```

Comment:
Grammatically, I would use `Just like` rather than `Same as` here.

Reply:
I tried `Just like` but found it a little confusing or casual.  I changed it to `Equivalent to`.  Let me know what you think.  Thanks for your feedback.

Reply:
👍